docs: add info about adding popups exception and remove jank on load (#210)
stackblitz/docs: docs/platform/webcontainers/browser-config.md

Read on if you run into issues or want an even smoother experience.

### Previews in a separate tab {#chrome-preview-separate-tab}

> Previews opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.

Reviewer: nitpick: 

"You usually get prompted to click a button to do that." -> "To do this, you are usually prompted to click a button."

> Previews that are opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.

Reviewer 2: ```suggestion
Previews opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.
```

> You can have it done automatically by modifying your popups settings, [chrome://settings/content/popups](chrome://settings/content/popups), with the following exception:

Reviewer: nitpick:

maybe something like: 

This can happen automatically if your popups settings are adjusted with the following extension:

```
https://[*.]webcontainer.io
```

<img class="mx-auto" alt="Chrome popups settings showing the *.webcontainer.io origins as exceptions" src="./assets/chrome-settings-popups.png" width="800" height="502.37" />

> Once done, whenever the preview need to reconnect to the editor, it will open a popup that gets immediately closed.

Reviewer 2: ```suggestion
Once done, whenever the preview needs to reconnect to the editor, it will open a popup that gets immediately closed.
```

> Lastly, if you have Chrome's [memory saver][CHROME_MEMORY_SAVER] turned on, this step might be neccessary more often than required. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):

Reviewer 2: ```suggestion
Lastly, if Chrome's [memory saver][CHROME_MEMORY_SAVER] is turned on, this step might be necessary more often than required. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):
```

Also, what do you mean by "might be necessary more often than required"?

<img
  alt="The Cookies and site data exceptions management window in Firefox settings. It shows a couple entries for stackblitz.io and webcontainer.io, both with the value “Allow”."
  src="./assets/firefox-settings-cookies-2.png"
  width="820"
  height="645"
/>

Don’t forget to click “Save Changes”. Then you should be able to reload the tab with your StackBlitz project, and hopefully everything should work!

### Previews in a separate tab {#firefox-preview-separate-tab}

> Previews opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.

Reviewer: Just noticed that the wording for firefox and chrome are different

> Previews that are opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.

Reviewer 2: ```suggestion
Previews opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.
```
